src/components/ExcelTest/ExcelTest.js

```diff
@@ -0,0 +1,178 @@
+import React, { Component } from "react";
```

Comment:
Might be worth changing the name and moving this component to the `Subcomponents` folder